src/CameraDeviceV4l2.cpp

```diff
@@ -150,8 +150,7 @@ int CameraDeviceV4l2::resetParams(CameraParameters &camParam)
             if (queryctrl.flags & V4L2_CTRL_FLAG_DISABLED)
                 continue;
         }
-        ret = v4l2_set_control(fd, queryctrl.id, queryctrl.default_value);
-        //    log_debug("return value for:%s : %d : %d",queryctrl.name,queryctrl.id,ret);
+        v4l2_set_control(fd, queryctrl.id, queryctrl.default_value);
```

Comment:
We need to check the return value, return failure even if one param reset fails

Reply:
@SaliniVenate Did you see this?

Reply:
Yes, I saw the comment and had an internal discussion with @lbegani. He will raise a PR which takes care of issues in this PR. So I'm closing this PR.